packages/client/hooks/useModal.ts

```diff
@@ -19,7 +19,8 @@ const useModal = (options: Options = {}) => {
     id,
     parentId,
     onOpen,
-    onClose
+    onClose,
+    allowScroll: false
```

Comment:
-1 Scrolling is broken on unrelated modals with your pr.
master:
![master](https://github.com/ParabolInc/parabol/assets/7331043/bfa8928f-355b-45d6-bdd2-10ed051d8f83)
this pr:
![pr](https://github.com/ParabolInc/parabol/assets/7331043/2818ebf7-22e3-4481-82fe-8e7ab02483b8)
Not sure if it's related to this line and I stopped checking after I found the first issue.